docs/sdks/kotlin-sdk/integrate-with-okta.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Integrate With Okta
+sidebar_position: 2
+---
+
+This guide describes how to configure Beyond Identity using Okta as the OIDC authentication flow.
```

Comment:
"This guide describes how to configure Okta to delegate to Beyond Identity for authentication during an OAuth2 authorization flow."

Reply:
Fixed